repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/gsheet/GoogleSpreadSheetFactory.java

```diff
@@ -33,8 +33,9 @@
     private Logger logger = LoggerFactory.getLogger(GoogleSpreadSheetFactory.class);
     private static final String APPLICATION_NAME = "RepairNator Bot";
     private static final File DATA_STORE_DIR = new File(System.getProperty("user.home"), ".credentials/sheets.googleapis.com-repairnator");
-    private static final String GOOGLE_SECRET_PATH = "./client_secret.json";
-    public static final String SPREADSHEET_ID = "1FUHOVx1Y3QZCAQpwcrnMzbpmMoWTUdNg0KBM3NVL_zA";
+    private static final String GOOGLE_SECRET_PATH = "/home/fernanda/git/librepair/repairnator/client_secret.json";
+    //public static final String SPREADSHEET_ID = "1FUHOVx1Y3QZCAQpwcrnMzbpmMoWTUdNg0KBM3NVL_zA";
+    public static final String SPREADSHEET_ID = "1MnRwoZGCxxbmkiswc0O6Rg43wJFTBc3bIyrNdTiBhQ4";
 
```

Comment:
Those lines should not be changed. We could use an argument for the `GOOGLE_SECRET_PATH` with a default value. And for `SPREADSHEET_ID` the constant is not used there but in the subclasses: we should either define them in subclasses or pass them as arguments.